### ISSUE TITLE
null response for paginator total and hasNextPage attributes.

### ISSUE THREAD

**Issue by vivek-mf** [eelcol/laravel-meilisearch#9]
Found a bug that when we are using pagination and query-ing `multi-search` endpoint, total and hasNestPage are null in response. Below are functions of `MeilisearchQueryCollection` class, for `multi-search` need to handle to get it from `results` attribute in response

```
public function totalCount(): ?int
    {
        if (!isset($this->result)) {
            return null;
        }
        return $this->result->json('totalHits') ?? $this->result->json('estimatedTotalHits');
    }

    public function hasNextPage(): bool
    {
        if (!$this->result->json('page') || !$this->result->json('totalPages')) {
            return false;
        }

        return $this->result->json('page') < $this->result->json('totalPages');
    }
```

**Comment by eelcol** [comment]
Can you give the code that you have made that produced this issue?

**Comment by vivek-mf** [comment]
```
$products = MeilisearchQuery::index('products');
$products->whereIn("brand.slug", ["tata","kia","maruti"]);
return $products->setFacets(['brand.slug'])->paginate(20);
```

`total` in paginate response is coming null

**Comment by eelcol** [comment]
Do you have an attribute in de index `products` named `brand.slug`? Or is the brand a separate index. Just to make sure, you can't use relations, every attribute you are searching on, must be in de index `products`.

**Comment by vivek-mf** [comment]
`brand.slug` is in products object itself.

You have already handled `multi-search` endpoint  of meilisearch response in `MeilisearchQueryCollection` class __construct on line 25  function but its not handled in `totalCount` and `hasNextPage` functions.

https://github.com/eelcol/laravel-meilisearch/blob/v2.x/src/Connector/Collections/MeilisearchQueryCollection.php#L25

**Comment by eelcol** [comment]
Alright thanks, I will look into it asap.

**Comment by eelcol** [comment]
As you mentioned, this issue exists when using the multi-search endpoint. I have created a fix in version `2.3.4`:

https://github.com/eelcol/laravel-meilisearch/commit/9c58cc66fe49a89a3a53a5a822c0bb6bc0302b4d
